### ISSUE TITLE
branch: accessibility-mixin

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/circuit-construction-kit-common#241]
In https://github.com/phetsims/scenery/issues/596 AccesssibleNode will be removed and replaced with a mixin for scenery/node called Accessibility.js.  This work is still being maintained on a branch in scenery.

This branch has the necessary changes for the transition from AccessibleNode to the mixin, and should be merged with master when the Accessibility mixin is merged into the scenery master branch.

**Comment by jessegreenberg** [comment]
Merge completed via a21782d40ff1659df73316968362a936711ad5d7. Deleting the branch.

**Comment by jessegreenberg** [comment]
Branch deleted. Closing.
